app/src/minimal/kotlin/io/homeassistant/companion/android/sensors/LocationProvider.kt

```diff
@@ -0,0 +1,50 @@
+package io.homeassistant.companion.android.sensors
```

Comment:
Why is this file here? It's not a sensor and there is already a class with the same name in the location package.

Reply:
It's a duplicate of the one in `location` package. I messed up while moving the classes between flavors. Fixing it.